components/ExportURLInput.tsx

```diff
@@ -1,8 +1,9 @@
 import { TextInput } from '@mantine/core';
-import { useState } from 'react';
+import { exportUrlState } from '../atoms/exportUrl';
+import { useRecoilState } from 'recoil';
 
 export default function ExportURLInput() {
-  const [exportURL, setExportURL] = useState<string>('');
+  const [exportURL, setExportURL] = useRecoilState<string>(exportUrlState);
```

Comment:
```suggestion
  const [exportURL, setExportURL] = useRecoilState(exportUrlState);
```

Shouldn't need this since the type will get inferred from the recoil atom